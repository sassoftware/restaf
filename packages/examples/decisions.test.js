let testFunctions = require("./testFunctions");
let testInfo;
beforeAll(async () => {
	testInfo = await require('./lib/setupAll')();
});
test("List repositories", async () => {
	expect.assertions();
	let l = await testFunctions.modelRepoRoot(testInfo);
	expect(l).toMatchSnapshot();
});
test('Create CAS Destination', async () => {
	expect.assertions();
	let l = await testFunctions.modelDestinationCas(testInfo);
	expect(l).toMatchSnapshot();
});
test.only('test mlPipeline', async () => {
	expect.assertions();
	let l = await testFunctions.mlPipeline(testInfo);
	expect(l).toMatchSnapshot();
});
