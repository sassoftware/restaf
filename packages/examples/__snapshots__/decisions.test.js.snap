// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create CAS Destination 1`] = `"done"`;

exports[`List repositories 1`] = `"done"`;

exports[`test mlPipeline 1`] = `"done"`;
